week-3/F-array-forEach/exercise.js

```diff
@@ -8,6 +8,17 @@
 */
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+arr.map(function isMultipleOf3(number) {
+  if (number % 3 == 0 && number != 0) {
+    console.log("Fizz");
+  } else if (number % 5 == 0 && number != 0) {
+    console.log("Buzz");
+  } else if (number % 3 == 0 && number % 5 == 0 && number != 0) {
```

Comment:
Have you tested this? Did you look carefully at the expected output below? Is it exactly the same as the output you see when you run this?

Remember that we compare the `if` statement first, then any `else if` statements in order.